packages/app/src/pages/dashboard/groups/_/body/add/index.tsx

```diff
@@ -63,28 +65,28 @@ const Add: React.FC<Props> = ({ onAdd, onCancel }) => {
         <div>
           <Head
             on={COLOR_SYSTEM.SURFACE}
-            title="Create a Group"
-            description="Manage endpoints in groups."
+            title={t('createGroup.head.title')}
+            description={t('createGroup.head.description')} //エンドポイントをグループで管理します。
```

Comment:
```suggestion
            description={t('createGroup.head.description')}
```

Reply:
@nonoakij  Sorry, I forgot to delete the comment out.